research-design.tex (#508)
worldbank/dime-data-handbook: appendix/research-design.tex

%-----------------------------------------------------------------------------------------------

\begin{fullwidth}

> Development Research in Practice focuses on tools, workflows, and practical 

Reviewer: ```suggestion
\textit{Development Research in Practice} focuses on tools, workflows, and practical 
```

guidance for implementing research projects. 
While not central to the content of this book, 

> we realize how essential it is for all research team members

Reviewer: ```suggestion
we think it is essential for all research team members
```

> - including field staff and research assistants - 

Reviewer: ```suggestion
-- including field staff and research assistants -- 
```

to understand research design, 

> and specifically how that research design impacts data work. 

Reviewer: ```suggestion
and specifically how research design choices impact data work. 
```

to obtain an understanding of the way in which each 
causal inference method constructs treatment and control groups,
the data structures needed to estimate the corresponding effects,
and specific code tools designed for each method.

It is essential for the research team members who will do the data work 
to understand the study design, for several reasons.
If you do not know how to calculate the correct estimator for your study,
you will not be able to assess the statistical power of your research design.
You will also be unable to make decisions in the field
when you inevitably have to allocate scarce resources
between tasks like maximizing sample size
and ensuring follow-up with specific individuals.
You will save time by understanding the way your data needs to be organized
to produce meaningful analytics throughout your projects.
Just as importantly, familiarity with each of these approaches
will allow you to keep your eyes open for research opportunities:
many of the most interesting projects occur because people in the field
recognize the opportunity to implement one of these methods
in response to an unexpected event.

This appendix is split into two sections.

> The first covers causal inference methods, 

Reviewer: ```suggestion
The first covers causal inference methods
```

if they had not been treated, and it is particularly effective at doing so
as evidenced by its broad credibility in fields ranging from clinical medicine to development.
Therefore RCTs are very popular tools for determining the causal impact
of specific programs or policy interventions, 
as evidenced by the awarding of the 2019 Nobel Prize in Economics
to Abhijit Banerjee, Esther Duflo and Michael Kremer 

> “for their experimental approach to alleviating global poverty”.\cite{nobel2019}

Reviewer: ```suggestion
``for their experimental approach to alleviating global poverty''.\cite{nobel2019}
```
